src/charactersheet/models/common/image.js

```diff
@@ -23,28 +24,52 @@ export class EncounterImage extends KOModel {
 
     DESCRIPTION_MAX_LENGTH = 100;
 
-    // Public Methods
-
-    toJSON = function() {
```

Comment:
what's the story here?

Reply:
These were for pCards and send-to-chat, which we don't use anymore.